app/src/main/java/com/android/shelfLife/ui/utils/DateUtils.kt

```diff
@@ -227,6 +227,17 @@ fun formatTimestampToDate(timestamp: Timestamp): String {
   return sdf.format(timestamp.toDate())
 }
 
+/**
+ * Function to format a Timestamp to a date string to be displayed (includes the slashes).
+ *
+ * @param timestamp The Timestamp to format.
+ * @return The formatted date string (stored as digits without slashes).
+ */
+fun formatTimestampToDisplayDate(timestamp: Timestamp): String {
+  val sdf = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
+  return sdf.format(timestamp.toDate())
+}
```

Comment:
Since there has been much talk about localisation and string resources in our meetings recently, you could allow a parameter to modify the format, which could be more inclusive for American users who may want to chose a different format option
```suggestion
fun formatTimestampToDisplayDate(timestamp: Timestamp, formatString="dd/MM/yyyy"): String {
  val sdf = SimpleDateFormat(formatString, Locale.getDefault())
  return sdf.format(timestamp.toDate())
}
```